archivist/src/test/scala/uk/ac/wellcome/platform/archive/archivist/bag/ZippedBagFileTest.scala

```diff
@@ -39,29 +39,6 @@ class ZippedBagFileTest
       }
     }
 
-    it("fails if there is no valid bag-info file in zip file") {
```

Comment:
This test fails in `master` and i've no idea why - so i'm blatting it as I don't feel like it's worth fixing.